engine/polyglot-api/src/main/scala/org/enso/polyglot/Suggestion.scala

```diff
@@ -110,6 +110,44 @@ object Suggestion {
       }
   }
 
+  @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
```

Comment:
This is not the API change described in the [current specification](https://github.com/enso-org/design/blob/76542b2a6c608f0de683aca1a6df8be6a7c1eb44/epics/widgets/drop-downs/design.md#architecture-overview). The change proposed here is bigger, not compatible change of existing `@Suggestion` protocol.

Reply:
right, fixed to conform to the spec